packages/oc-s3-storage-adapter/index.js

```diff
@@ -27,25 +27,52 @@ module.exports = function(conf) {
     return true;
   };
 
+  // Defaults
+  const bucket = conf.bucket ? conf.bucket : '';
+  const sslEnabled = conf.sslEnabled === false ? false : true;
+  const s3ForcePathStyle = conf.s3ForcePathStyle ? true : false;
+  const signatureVersion = conf.signatureVersion
+    ? conf.signatureVersion
+    : AWS.Config.signatureVersion;
   const httpOptions = { timeout: conf.timeout || 10000 };
   if (conf.agentProxy) {
     httpOptions.agent = conf.agentProxy;
   }
 
-  AWS.config.update({
+  // Setup AWS config
+  let awsConfig = new AWS.Config({
     accessKeyId: conf.key,
     secretAccessKey: conf.secret,
-    region: conf.region,
+    signatureVersion: signatureVersion,
```

Comment:
Perhaps we could use [Shorthand property names](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer) here

Reply:
Like this?

```
  const
    bucket = conf.bucket ? conf.bucket : '',
    sslEnabled = conf.sslEnabled === false ? false : true,
    s3ForcePathStyle = conf.s3ForcePathStyle ? true : false,
    signatureVersion = conf.signatureVersion ? conf.signatureVersion : AWS.Config.signatureVersion,
    httpOptions = { timeout: conf.timeout || 10000 };
```

Reply:
Not really, not a big fun of coma separated assignements as it also get in the way when refactoring of working on it.

By `Shorthand property names` I mean:

```
const signatureVersion = ....
const s3ForcePathStyle = ...
const sslEnabled = ....


let awsConfig = new AWS.Config({
signatureVersion,
s3ForcePathStyle,
sslEnabled
)}
...
```